### ISSUE TITLE
TRAFFIC MANAGER - level 60 can unblock level 100

### ISSUE THREAD

**Issue by Sopor** [luadch/luadch#36]
In `cfg.tbl` i can set `etc_trafficmanager.lua` and what level that can block other levels.
If a user with level 100 block a user with level 80, this level 80 user can unblock himself even if this level 80 user only can block users with level 70 and below. Even a level 60 can unblock a level 80 or even a level 100.
So all levels that can block other users (level 60-100) can unblock any user from level 0-100.
I can even block and unblock myself 😯 
A user with level 100 should not be able to block another user with level 100 and of course should a level 60 not be able to unblock higher levels than is set in cfg.tbl `etc_trafficmanager.lua`.
It would be nice if i could set the unblock levels the same way i now set the block levels. Then it is possible for the hub owners to choose what levels that can unblock.

Luadch 2.19

**Comment by Sopor** [comment]
Now i can't unblock a user. It will show `unknown command` and it will show this error message:
```
 scripts.lua: script error: ././scripts/etc_trafficmanager.lua:675: attempt to read undeclared var: 'target' (listener: onBroadcast; script: 'etc_hubcommands.lua')
```

**Comment by Sopor** [comment]
@blasti Should i create a new issue or what?
